index.html

```diff
@@ -10,229 +10,233 @@
+                    don't hestiate to contact me.
+                </p>
+            </div>
+
+            <ul class="network">
+                <li class="social">
+                    <a target="_blank" href="https://github.com/Kakalanp" rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/github.png" alt="g"/></a>
+                </li>
+                <li class="social">
+                    <a target="_blank" href="https://www.linkedin.com/in/joaquin-garrido-lecca-zanetti-623583204/" rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/linkedin-2--v1.png" alt="li"/></a>
+                </li>
+                <li class="social">
+                    <a target="_blank" href="https://www.twitter.com/LeccaJoaquin"  rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/twitter--v1.png" alt="t"/></a>
+                </li>
+            </ul>
         </section>
 
-        <section class="card">
-            <section class="screenshot">
-                <img src="/Images/placeholder.png" alt="screenshot">
-            </section>
-            
-            <section class="info">
-                <h3 class="text">
-                    Multi-Post Stories Gain+Glory
-                </h3>
-                <ul class="languages">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>Javascript</li>
-                </ul>
-                <a class="button" href="">See project</a>
-            </section>
-        </section>
 
-        <section class="card">
-            <section class="screenshot">
-                <img src="/Images/placeholder.png" alt="screenshot">
+        <section class="works">
+            <section id="works-title">
+                <h2>My recent works</h2>
+                <img src="/Images/works-horizontal-line.svg" alt="line">
             </section>
             
-            <section class="info">
-                <h3 class="text">
-                    Multi-Post Stories Gain+Glory
-                </h3>
-                <ul class="languages">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                </ul>
-                <a class="button" href="">See project</a>
-            </section>
-        </section>
+            <div class="works">
+                <section class="card">
+                    <section class="screenshot">
+                        <img src="/Images/placeholder.png" alt="screenshot">
+                    </section>
+                    
+                    <section class="info">
+                        <h3 class="text">
+                            Multi-Post Stories Gain+Glory
+                        </h3>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                        </ul>
+                        <a class="button" href="">See project</a>
+                    </section>
```

Comment:
- [ ] These cards are well placed. By the look of things, you included much space between the contents and the button. I will suggest you reduce the space.

Your Design | Figma Design  |
:---------------:|:--------------------:|
![e3](https://user-images.githubusercontent.com/39523312/149404294-94c28dec-e237-459a-88e1-d52f5f173790.png) |![e4](https://user-images.githubusercontent.com/39523312/149404308-1ea3eb4a-6b5a-40ce-981d-6ac020a41bfc.png)

Reply:
You are pointing that out either in the 922px version or way more than 375px (as recommended for mobile), that's exactly why it looks that way.
In the full desktop version, it still looks different from Figma because the Figma version is a 1440px version and this isn't.